tests/DifferTest.php

```diff
@@ -475,19 +475,21 @@ public function textForNoNonDiffLinesProvider(): array
         ];
     }
 
+    /**
+     * @expectedException \InvalidArgumentException
+     * @expectedExceptionMessageRegExp #^"from" must be an array or string\.$#
+     */
     public function testDiffToArrayInvalidFromType()
     {
-        $this->expectException('\InvalidArgumentException');
-        $this->expectExceptionMessageRegExp('#^"from" must be an array or string\.$#');
-
         $this->differ->diffToArray(null, '');
     }
 
+    /**
+     * @expectedException \InvalidArgumentException
+     * @expectedExceptionMessageRegExp #^"to" must be an array or string\.$#
+     */
     public function testDiffInvalidToType()
     {
-        $this->expectException('\InvalidArgumentException');
-        $this->expectExceptionMessageRegExp('#^"to" must be an array or string\.$#');
```

Comment:
I think it's available in phpunit5, isn't it?
then, we could still use it as we do php 5.6+